web/cgi-bin/horas/specials.pl

```diff
@@ -16,8 +16,11 @@ sub specials {
   my $lang = shift;		 
   
   $octavam = ''; #check duplicate commemorations
-  my %w = (columnsel($lang)) ? %winner : %winner2; 
-  
+  my %w = (columnsel($lang)) ? %winner : %winner2;
+
+  my $hymntrans = $translate{$lang}{Hymnus};
```

Comment:
This always gives `'Hymnus'`
If you mean
```
my $hymntrans = $translate{$lang}{Hymnus} || 'Hymnus';
```
it is not necessery as translate always contains default Latin values